README.md

```diff
@@ -1,76 +1,65 @@
 jq
 ==
 
-jq is a lightweight and flexible command-line JSON processor.
+## Introduction
 
-If you want to learn to use jq, read the documentation at
-[https://jqlang.github.io/jq](https://jqlang.github.io/jq).  This
-documentation is generated from the docs/ folder of this repository.
-You can also try it online at [jqplay.org](https://jqplay.org).
+**jq** is a lightweight and flexible command-line JSON processor akin to `sed`, `awk`, `grep`, and friends for JSON data. It's written in portable C and has zero runtime dependencies, allowing you to easily slice, filter, map, and transform structured data.
 
-If you want to hack on jq, feel free, but be warned that its internals
-are not well-documented at the moment. Bring a hard hat and a
-shovel.  Also, read the wiki: https://github.com/jqlang/jq/wiki, where
-you will find cookbooks, discussion of advanced topics, internals,
-release engineering, and more.
+## Getting Started
 
-Source tarball and built executable releases can be found on the
-homepage and on the github release page, https://github.com/jqlang/jq/releases.
-Docker image is available at https://github.com/jqlang/jq/pkgs/container/jq.
+### Documentation
 
-If you're building directly from the latest git, you'll need libtool, make, automake,
-and autoconf installed.
-To get regexp support you'll also need to install Oniguruma or clone it as a
-git submodule as per the instructions below. To build, run:
+- **Official Documentation**: [jqlang.github.io/jq](https://jqlang.github.io/jq)
+- **Try jq Online**: [jqplay.org](https://jqplay.org)
 
-    git submodule update --init # if building from git to get oniguruma
-    autoreconf -i               # if building from git
-    ./configure --with-oniguruma=builtin
-    make -j8
-    make check
+## Installation
 
-Developers must `--enable-maintainer-mode` when making changes to the
-jq parser and lexer which also requires bison and flex to be installed.
+### Prebuilt Binaries
 
-To build a statically linked version of jq, run:
+Download the latest releases from the [GitHub release page](https://github.com/jqlang/jq/releases).
 
-    make LDFLAGS=-all-static
+### Docker Image
 
-After make finishes, you'll be able to use `./jq`.  You can also
-install it using:
+Pull the [jq image](https://github.com/jqlang/jq/pkgs/container/jq) to start quickly with Docker.
 
-    sudo make install
+### Building from source
 
-If you're not using the latest git version but instead building a
```

Comment:
I feel like we're losing a lot of detail -- detail that probably doesn't belong in this README, but it belongs somewhere.

Reply:
I put back this `if you're not ...` line to avoid confusions, the detail wasn't lost, it was made more concise.